recipe/meta.yaml

```diff
@@ -23,16 +23,17 @@ requirements:
     - make   # [unix]
     - xeus =3.0.5       # [build_platform != target_platform]
     - xeus-zmq =1.0.2   # [build_platform != target_platform]
-    - llvmdev =15.0.7   # [build_platform != target_platform]
+    - llvmdev =16.0.6   # [build_platform != target_platform]
     - xtl               # [build_platform != target_platform]
     - nlohmann_json     # [build_platform != target_platform]
     - cppzmq            # [build_platform != target_platform]
     - zlib              # [build_platform != target_platform]
+    - zstd-static =1.5.5  # [build_platform != target_platform]
```

Comment:
Why use the static library?

Reply:
I wrote up our reasons here: https://github.com/conda-forge/lfortran-feedstock/pull/48#issuecomment-1753237103.